blenderproc/python/utility/Initializer.py

```diff
@@ -1,18 +1,36 @@
 """ This module provides functions to init a BlenderProc scene. """
 
+import atexit
 import os
 import random
+import signal
+import shutil
 
 from numpy import random as np_random
 import bpy
 
 from blenderproc.python.utility.GlobalStorage import GlobalStorage
 from blenderproc.python.utility.Utility import reset_keyframes
+from blenderproc.python.utility.SetupUtility import SetupUtility, is_using_external_bpy_module
 from blenderproc.python.camera import CameraUtility
 from blenderproc.python.utility.DefaultConfig import DefaultConfig
 from blenderproc.python.renderer import RendererUtility
 
 
+def clean_temp_dir():
+    # pylint: disable=import-outside-toplevel
+    from blenderproc.python.utility.Utility import Utility
+    # pylint: enable=import-outside-toplevel
+    assert is_using_external_bpy_module()
+    if os.path.exists(Utility.temp_dir):
+        print("Cleaning temporary directory")
+        shutil.rmtree(Utility.temp_dir)
+
+
+def handle_sigterm(_signum, _frame):
+    clean_temp_dir()
+
+
 def init(clean_up_scene: bool = True):
```

Comment:
Can we add `temp_dir` here as additional parameter, such that it can be changed by the user in external mode?

Reply:
Done, makes sense, now the options should be same for both modes.